dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md

```diff
@@ -0,0 +1,300 @@
+
+    ```
+    NAME                                       READY     STATUS    RESTARTS   AGE
+    tidb-controller-manager-5cd94748c7-jlvfs   1/1       Running   0          1m
+    tidb-scheduler-56757c896c-clzdg            2/2       Running   0          1m
+    ```
+
+## 第 3 步: 在 kind Kubernetes 集群中部署 TiDB 集群
+
+通过 `helm` 和 TiDB Operator，我们可以很轻松的部署一套 TiDB 集群：
+
+{{< copyable "shell-regular" >}}
+
+``` shell
+helm install pingcap/tidb-cluster --name=demo --namespace=tidb --set pd.storageClassName=<storageClass> --set tikv.storageClassName=<storageClass> --version=<chartVersion>
+```
+
+等待几分钟，确保 TiDB 所有组件正常创建并进入 `ready` 状态，可以通过下面命令持续观察：
+
+{{< copyable "shell-regular" >}}
+
+``` shell
+kubectl get pods --namespace tidb -l app.kubernetes.io/instance=demo -o wide --watch
+```
+
+当所有 Pod 状态为 `Running`，<kbd>Ctrl</kbd>+<kbd>C</kbd>  停止 watch。
+
+通过下面步骤获取集群信息：
+
+{{< copyable "shell-regular" >}}
+
+``` shell
+kubectl get tidbcluster -n tidb
+```
+
+输出类似如下信息：
+
+```
+NAME   PD                       STORAGE   READY   DESIRE   TIKV                       STORAGE   READY   DESIRE   TIDB                       READY   DESIRE
+demo   pingcap/pd:v3.0.0-rc.1   1Gi       3       3        pingcap/tikv:v3.0.0-rc.1   10Gi      3       3        pingcap/tidb:v3.0.0-rc.1   2       2
+```
+
+{{< copyable "shell-regular" >}}
+
+``` shell
+kubectl get statefulset -n tidb
+```
+
+输出类似如下信息：
+
+```
+NAME        DESIRED   CURRENT   AGE
+demo-pd     3         3         1m
+demo-tidb   2         2         1m
+demo-tikv   3         3         1m
+```
+
+{{< copyable "shell-regular" >}}
+
+``` shell
+kubectl get service -n tidb
+```
+
+输出类似如下信息：
+
+```
+NAME              TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)                          AGE
+demo-discovery    ClusterIP   10.96.146.139    <none>        10261/TCP                        1m
+demo-grafana      NodePort    10.111.80.73     <none>        3000:32503/TCP                   1m
+demo-pd           ClusterIP   10.110.192.154   <none>        2379/TCP                         1m
+demo-pd-peer      ClusterIP   None             <none>        2380/TCP                         1m
+demo-prometheus   NodePort    10.104.97.84     <none>        9090:32448/TCP                   1m
+demo-tidb         NodePort    10.102.165.13    <none>        4000:32714/TCP,10080:32680/TCP   1m
+demo-tidb-peer    ClusterIP   None             <none>        10080/TCP                        1m
+demo-tikv-peer    ClusterIP   None             <none>        20160/TCP                        1m
+```
+
+{{< copyable "shell-regular" >}}
+
+``` shell
+kubectl get configmap -n tidb
+```
+
+输出类似如下信息：
+
+```
+NAME                              DATA   AGE
+demo-monitor                      5      1m
+demo-monitor-dashboard-extra-v3   2      1m
+demo-monitor-dashboard-v2         5      1m
+demo-monitor-dashboard-v3         5      1m
+demo-pd                           2      1m
+demo-tidb                         2      1m
+demo-tikv                         2      1m
```

Comment:
From tidb-cluster chart v1.0.0, configmap rollout feature is enabled by default, there should be cm with hash values for pd, tidb and tikv.